tests/native_api_tests/test_api_model.cpp

```diff
@@ -372,16 +390,6 @@ TEST_CASE("API Model") {
             CHECK(batch_node_result_u[1] == doctest::Approx(0.0));
             CHECK(batch_node_result_u_pu[1] == doctest::Approx(0.0));
             CHECK(batch_node_result_u_angle[1] == doctest::Approx(0.0));
-            CHECK(batch_node_result_id[2] == 0);
-            CHECK(batch_node_result_energized[2] == 1);
-            CHECK(batch_node_result_u[2] == doctest::Approx(70.0));
-            CHECK(batch_node_result_u_pu[2] == doctest::Approx(0.7));
-            CHECK(batch_node_result_u_angle[2] == doctest::Approx(0.0));
-            CHECK(batch_node_result_id[3] == 4);
-            CHECK(batch_node_result_energized[3] == 0);
-            CHECK(batch_node_result_u[3] == doctest::Approx(0.0));
-            CHECK(batch_node_result_u_pu[3] == doctest::Approx(0.0));
-            CHECK(batch_node_result_u_angle[3] == doctest::Approx(0.0));
```

Comment:
This was removed because the second update (`*[2]` and `*[3]`) is expected to "fail" as the updated id is wrong.